assets/js/blocks/products-by-attribute/block.js

```diff
@@ -171,7 +171,7 @@ class ProductsByAttributeBlock extends Component {
 							{
 								icon: 'edit',
 								title: __(
-									'Edit',
+									'Edit selected attributes',
```

Comment:
For this block, we can select one attribute only.
```suggestion
									'Edit selected attribute',
```